modelzoo/DeepFM/train.py

```diff
@@ -210,122 +171,267 @@ def prediction(self):
         all_input = tf.concat([dnn_output, linear_output, fm_output], 1)
         final_dnn_scope = tf.variable_scope('final_dnn')
         with final_dnn_scope.keep_weights(dtype=tf.float32) if self.bf16 \
-            else final_dnn_scope:
-            net = self.dnn(all_input, self.final_hidden_units, 'final_dnn')
+                else final_dnn_scope:
+            net = self._dnn(all_input, self._final_hidden_units, 'final_dnn')
 
         if self.bf16:
             net = tf.cast(net, dtype=tf.float32)
 
         net = tf.layers.dense(net, units=1)
-        net = tf.math.sigmoid(net)
+        self.predict = tf.math.sigmoid(net)
```

Comment:
self._logits

Reply:
@chenzhiyin  Since our customer is waiting for the update, please help to modify the code.  If any help is needed, pls let us know. Thanks.

Reply:
OK, I am working on it.